packages/dds/tree/src/tree/cursor.ts

```diff
@@ -96,6 +96,19 @@ export interface ITreeCursor {
      */
     enterNode(childIndex: number): void;
 
+    /**
+     * @returns a path to the current node.
```

Comment:
Minor suggestion: it could be useful to add an example of what the path should look like.

Reply:
I have added a link to FieldUpPath which defines this format. Putting example object literals inline here would be annoying to maintain if changing the FieldUpPath, so I think its better to just refer to it.